runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/GlusterFileStore.java

```diff
@@ -101,7 +101,7 @@ public void writeBlock(final Block block) throws BlockWriteException {
    * @throws BlockFetchException for any error occurred while trying to fetch a block.
```

Comment:
Would you please remove the stale comment in L101?

Reply:
It's done!